### ISSUE TITLE
Openconfig configuration on Juniper MX does not work

### ISSUE THREAD

**Issue by krihal** [clicon/clixon-controller#20]
When connection to Juniper MX:

```
khn@zoomie> show devices
Name                    State      Time                   Logmsg
=======================================================================================
orval                   CLOSED     2023-08-10T08:32:58    YANG binding failed at mountp
khn@zoomie>
```

**Comment by olofhagsand** [comment]
The error seems to be an openconfig network-instances configuration on the top-level of the MX. 
The openconfig yang is not announced by the MX and is also a separate(second) top-level root of the configuration.
This means that the opeconfig needs to be loaded and mounted separately, in some other mechanism than the regular.

**Comment by pheller** [comment]
This might be a reason to consider https://github.com/clicon/clixon-controller/issues/21

**Comment by olofhagsand** [comment]
Note the "fixed ref" ^ is a mistake, should be #19

**Comment by olofhagsand** [comment]
This should now be possible to do using device profiles and adding local module-sets for openconfig
